server/internal/core/application/covenantless.go

```diff
@@ -1401,6 +1401,21 @@ func (s *covenantlessService) finalizeRound(notes []note.Note) {
 		return
 	}
 
+	// TODO: make this concurrent and ensure all forfeits are verified before returning error
```

Comment:
`verifyTapscriptPartialSigsMap(txs map[string]*psbt.Packet)` let's reuse that function ?

Reply:
done